policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/work_premium/general_work_premium/couple/amount/with_children.yaml

```
@@ -0,0 +1,12 @@
+description: Quebec limits its work premium credit to eligible couple with at least one child at or below this amount.
```

Similarly elsewhere - we use `limit` to refer to eligibility criteria
```suggestion
description: Quebec provides a work premium credit up to this amount to eligible couples with at least one child.
```